packages/ui/components/ui/tx/view/memo-view.tsx

```diff
@@ -1,20 +1,12 @@
 import { MemoView } from '@buf/penumbra-zone_penumbra.bufbuild_es/penumbra/core/transaction/v1alpha1/transaction_pb';
 import { AddressViewComponent } from './address-view';
 import { ViewBox, ViewSection } from './viewbox';
-import { AddressView } from '@buf/penumbra-zone_penumbra.bufbuild_es/penumbra/core/keys/v1alpha1/keys_pb';
 
 export const MemoViewComponent = ({ memo }: { memo: MemoView }) => {
   if (memo.memoView.case === 'visible') {
     const mv = memo.memoView.value;
     const text = mv.plaintext?.text;
 
-    const av = new AddressView({
```

Comment:
For some reason, `MemoViewComponent` was hard-coding the sender return address as opaque, rather than simply passing the memo's `returnAddress` property to `AddressViewComponent` and allowing `AddressViewComponent` to determine whether it was visible or opaque.

So I changed that logic here. As a result, the sender return address won't be opaque in the txn approval popup _or_ in the `TransactionViewComponent` for already-existing transactions:
![image](https://github.com/penumbra-zone/web/assets/1121544/3130c013-bc9a-43e3-aefb-a7a6bfee5788)